api-doc/openapi.yaml

```diff
@@ -1,48 +1,514 @@
+    DonationsOk:
+      description: Donation by id
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Donation'
+    GameDataAllOk:
+      description: List of game data
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/GameDataAll'
+    GameDataOk:
+      description: Game data by id
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/GameData'
+    StreamMetadataOk:
+      description: Stream metadata
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/StreamMetadata'
+    PlayerInfoOk:
+      description: Player info
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/PlayerInfo'
+    IncentivesAllOk:
+      description: List of incentives
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Incentives'
+    IncentivesOk:
+      description: Incentives by id (uuid)
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/Target'
+            $ref: '#/components/schemas/Incentive'
+    IncentiveCodesAllOk:
+      description: List of incentive code data
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/IncentiveCodes'
+    IncentiveCodesOk:
+      description: Incentice data by code
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/IncentiveCode'
   schemas:
-    Target:
-      type: object
-      required:
-        - target
-        - currency
-        - _links
-      properties:
-        _links:
-          type: object
-          required:
-            - href
-          properties:
-            href:
-              type: string
-              format: uri
-              description: A link to self
-        target:
+    Donation:
+      type: object
+      properties:
+        timestamp:
+          type: string
+        name:
+          type: string
+        message:
+          type: string
+        amount:
+          type: integer
+      required:
+      - timestamp
+      - message
+      - amount
+      example:
+        timestamp: '2021-09-21 15:05:47'
+        name: jsloth
+        message: ota tää :D 🤔🤔
+        amount: 10
+    Donations:
+      type: object
+      properties:
+        id:
+          $ref: '#/components/schemas/Donation'
+      required:
+      - id
+      example:
+        6c28df04-b9d2-4ebe-bd33-58c9b290d929:
+          timestamp: '2021-09-21 15:05:47'
+          name: jsloth
+          message: ota tää :D 🤔🤔
+          amount: 10
+        7c28df04-b9d2-4ebe-bd33-58c9b290d930:
+          timestamp: '2021-09-21 15:05:50'
+          name: luuranki
+          message: 'mee tänne: töihin :))))))))'
+          amount: 5
+    GameData:
```

Comment:
Separate player object would be nice, but I guess this will work as well. Does UI require separate info if there are multiple players

Reply:
Before if there has been multiple players it has been simply a string like "Glukoosi and T3mu" as far as I know. If you have more specific suggestion for player object please give an example.

Reply:
Then lets go with this, at least for now. It's simple and it gets the job done